libs/site-semantic-theme/src/semantic/thecoin/elements/input.overrides

```diff
@@ -8,12 +8,12 @@
 // definition could override this one
 .ui.input {
 
-  & > input {
+  & > input[type="text"] {
     font-weight: @bold;
 
     &:hover {
        border-color: @hoverBorderColor;
-       box-shadow: @hoverBoxShadow !important;
```

Comment:
important overrode the shadows on our range input, so had to apply it a different way